lighthouse-core/lib/emulation.js

```diff
@@ -64,40 +26,31 @@ const NO_CPU_THROTTLE_METRICS = {
   rate: 1,
 };
 
-const emulationParams = {
-  mobile: {
-    userAgent: MOTOG4_USERAGENT,
-    metrics: MOTOG4_EMULATION_METRICS,
-    touchEnabled: true,
-  },
-  desktop: {
-    userAgent: DESKTOP_USERAGENT,
-    metrics: DESKTOP_EMULATION_METRICS,
-    touchEnabled: false,
-  },
-};
-
 /**
  *
  * @param {Driver} driver
  * @param {LH.Config.Settings} settings
  * @return {Promise<void>}
  */
 async function emulate(driver, settings) {
-  if (!settings.emulatedFormFactor || settings.emulatedFormFactor === 'none') return;
-  const params = emulationParams[settings.emulatedFormFactor];
-
   // In DevTools, emulation is applied before Lighthouse starts (to deal with viewport emulation bugs)
-  // As a result, we don't double-apply viewport emulation (devtools sets `internalDisableDeviceScreenEmulation`).
-  // UA emulation, however, is lost in the protocol handover from devtools frontend to the audits_worker. So it's always applied.
-
-  // Network.enable must be called for UA overriding to work
-  await driver.sendCommand('Network.enable');
-  await driver.sendCommand('Network.setUserAgentOverride', {userAgent: params.userAgent});
-
-  if (!settings.internalDisableDeviceScreenEmulation) {
-    await driver.sendCommand('Emulation.setDeviceMetricsOverride', params.metrics);
-    await driver.sendCommand('Emulation.setTouchEmulationEnabled', {enabled: params.touchEnabled});
+  // As a result, we don't double-apply viewport emulation (devtools sets `screenEmulation` to `false`).
+  // UA emulation, however, is lost in the protocol handover from devtools frontend to the lighthouse_worker. So it's always applied.
+
+  if (settings.emulatedUserAgent !== false) {
+    // Network.enable must be called for UA overriding to work
+    await driver.sendCommand('Network.enable');
+    await driver.sendCommand('Network.setUserAgentOverride', {
+      userAgent: /** @type {string} */ (settings.emulatedUserAgent),
```

Comment:
having some trouble with this one.

Reply:
> having some trouble with this one.

yeah, I couldn't do it either without running right into `merge()` which would be bad news :)

Since it'll be purely internal (it's what `Config` spits out, not what it takes in), in theory if we figure out a nice way to do it we can add it later. If not, it's also not a huge issue.